share/empty.go

```diff
@@ -60,6 +67,10 @@ func EmptyExtendedDataSquare() *rsmt2d.ExtendedDataSquare {
 }
 
 // emptyDataSquare returns the minimum size data square filled with tail padding.
-func emptyDataSquare() [][]byte {
-	return shares.ToBytes(shares.TailPaddingShares(appconsts.MinShareCount))
+func emptyDataSquare() ([][]byte, error) {
+	result, err := shares.TailPaddingShares(appconsts.MinShareCount)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
If we panic above anyway and sure that this func always succeeds with given input, we can just panic here avoiding error return as it was